### PR TITLE
#4204 Updated for server details to not show for mitm.it

### PR DESCRIPTION
**Pull request by Samhita-sopho**
#### Description

Server section does not render if server address is null ( as seen in mitmproxy ) . Added a tflow with null server address to check if that fixes the code coverage issue as in the previous PR

#### Checklist

 - [x] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

### CODE REVIEW

**Comment by Kriechi** [comment]
I think you are missing some commit in this PR - I only see the updated changelog but no actual code changes...
ref #4204

File: CHANGELOG [hunk 1]
```diff
@@ -2,6 +2,7 @@ Unreleased: mitmproxy next
 
 ** Full Changelog **
 
+    * UI appears blank in mitmweb for mitm.it flow(@samhita-sopho)
```
Comment (Kriechi) [inline CHANGELOG:5]: Please rephrase to something "that you fixed".
"fixed xyz" or "added feature xyz"

Comment (Samhita-sopho) [reply CHANGELOG]: Thank you for pointing it , after updating the same I found that the test for web ui fails so working on it

**Comment by Samhita-sopho** [comment]
@Kriechi I have tested with npm test on my local machine and have seen this output 
Test Suites: 59 passed, 59 total
Tests:       316 passed, 316 total
Snapshots:   103 passed, 103 total
Time:        4.856s , 
**Summary of changes**: Made a null check to details.jsx and added a test flow with different parameters(_tflowServerAddressNull.js) , added a snapshot to test against and updated detailsSpec.js as well. Could you please let me know there is something else that needs to be done here

**Comment by mhils** [comment]
This seems to be a somewhat chaotic duplicate of #4224?

Closing this here for now.

**Pull request closed without merging**